website/docs/commands.md

```diff
@@ -18,3 +18,10 @@ window containing Neovide to the front and activate it. This
 is useful for tools like neovim_remote which can manipulate
 neovim remotely or if long running tasks would like to
 activate the Neovide window after finishing.
+
+## Open Config File
```

Comment:
This should probably indicate that this isn't released

Reply:
done